commands/latency-histogram.md

```diff
@@ -0,0 +1,37 @@
+The `LATENCY HISTOGRAM` command reports a cumulative distribution of latencies in the format of an histogram for each of the specified command names. 
+If no command names are specified then all commands that contain latency information will be replied.
+
+Each reported histogram has the following fields:
+
+* Command name.
+* The total calls for that command.
+* A map of time buckets:
+  * Each bucket represents a latency range.
+  * Each bucket covers twice the previous bucket's range.
+  * Empty buckets are not printed.
+  * The tracked latencies are between 1 microsecond and roughly 1 second.
+  * Everything above 1 sec is considered +Inf.
+  * At max there will be log2(1000000000)=30 buckets.
+
+This command requires the extended latency monitoring feature to be enabled (by default is enabled).
+If you need to enable it, use 'CONFIG SET latency-tracking yes'.
+
+@examples
+
+```
+127.0.0.1:6379> LATENCY HISTOGRAM set
+1# "set" => 1# "calls" => (integer) 100000
+   2# "histogram_usec" => 1# (integer) 1 => (integer) 99583
+      2# (integer) 2 => (integer) 99852
```

Comment:
Is this how redis-cli displays nested maps? (Maybe we need to do something about it in a different PR. It looks like it's misaligned in some way.)

Reply:
yes. it's indeed ugly.
thing is that we don't wanna add a newline if the map contains primitive, just when it contains a compound object.
feel free to look into if if you happen to have time.

Reply:
Candy: redis/redis#10170

Reply:
thanks.
can you please update this PR with the improved version.
and then merge it ;)

Reply:
```suggestion
127.0.0.1:6379> LATENCY HISTOGRAM set
1# "set" =>
   1# "calls" => (integer) 100000
   2# "histogram_usec" =>
      1# (integer) 1 => (integer) 99583
      2# (integer) 2 => (integer) 99852
```